packages/itwin/grouping-mapping-widget/src/widget/components/CalculatedPropertyTable.tsx

```diff
@@ -1,7 +1,3 @@
-/*---------------------------------------------------------------------------------------------
```

Comment:
revert this deletion
just needed to rvert the spacing changes

Reply:
done